### PR TITLE
Update bug report template by using githubs new issue forms

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    attributes:
+      label: "Installation"
+      description: "How did you install neovim: build from repo / system package manager / appimage / homebrew / snap / chocolatey / other (please specify)?"
+      placeholder: "Arch User Repository (AUR)"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: "Reproduce"
```
Comment (clason) [inline .github/ISSUE_TEMPLATE/bug_report.yml:45]: Maybe the old "Steps to reproduce" would be clearer.

Comment (dundargoc) [reply .github/ISSUE_TEMPLATE/bug_report.yml:45]: Did you mean that in the "label" field  I should replace the string "Reproduce" with "Steps to reproduce"? Or did you mean I should change back the whole section like it was previously?

Comment (clason) [reply .github/ISSUE_TEMPLATE/bug_report.yml:45]: The first: just the label.

Comment (dundargoc) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: fixed

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+name: Bug Report
+description: Report a problem in Neovim
+labels: [bug]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
```
Comment (clason) [inline .github/ISSUE_TEMPLATE/bug_report.yml:9]: "Thank you"

Comment (clason) [reply .github/ISSUE_TEMPLATE/bug_report.yml:9]: (etc.; the comments on the LSP PR apply here as well -- let me know if I should repeat them here)

**Comment by clason** [comment]
Thanks! Let's give it a try and see what happens :) I think this is going to be a big quality of life boost.

**Pull request merged**